source/lmp/pair_deepmd.cpp

```diff
@@ -450,9 +450,9 @@ void PairDeepMD::compute(int eflag, int vflag) {
   if (numb_models == 0) {
     return;
   }
-  if (eflag || vflag) {
-    ev_setup(eflag, vflag);
-  }
+  // See
+  // https://docs.lammps.org/Developer_updating.html#use-ev-init-to-initialize-variables-derived-from-eflag-and-vflag
+  ev_init(eflag, vflag);
```

Comment:
Consider adding a test case to verify that the `evflag` initialization issue is indeed fixed. This will help prevent potential regressions in the future. [important]